docs/oss_rules_of_play_rating.md

```diff
@@ -0,0 +1,65 @@
+# Rules of play ratings for open source projects
```

Comment:
Also is there some introduction on what `Rule of Play` is?

Reply:
Explanation now comes straight after the header as you proposed it.